addons/mail/static/src/models/file_uploader.js

```diff
@@ -213,7 +213,6 @@ registerModel({
                 }
                 return clear();
             },
-            required: true,
```

Comment:
How do we reach a case with no thread? I'm afraid in that case if the upload is attempted the attachment will not be linked to the thread correctly.

Reply:
I don't think this change is necessary again. I did that in a former version of the fix. Let's see whether runbot is green when I put this constraint back.

Reply:
Looks like this is still needed.
From what I can see, this happens only during a small frame when saving the record of chatter.
In practice, it's not possible to upload attachments on a temporary thread. This `required` is too strict and makes it crash, so it's best to just remove it than start making bigger modelling changes in `16.0`.